content/sql/concepts/commands/terms/union/union.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'UNION'
+Description: 'Allows the combination of the results of two or more SELECT queries.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Database'
+  - 'Queries'
+  - 'PostgreSQL'
+  - 'MySQL'
+  - 'SQLite'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+  - 'paths/design-databases-with-postgresql'
+---
+
+The `UNION` command allows for the combination of the results of two or more [`SELECT`](https://www.codecademy.com/resources/docs/sql/commands/select) queries. When two queries with the same number of columns are joined together with the `UNION` command, the result is a combination of the returned result from both queries. `UNION` is placed between the two queries being combined.
+
+The combined queries can be any valid `SELECT` statements, including [`WHERE`](https://www.codecademy.com/resources/docs/sql/commands/where), [`GROUP BY`](https://www.codecademy.com/resources/docs/sql/commands/group-by), or [`HAVING`](https://www.codecademy.com/resources/docs/sql/commands/having) clauses. The only restriction is that the number of columns returned for each query is the same, and the corresponding columns of each query have data types that are compatible with the columns in the first query. The result set will use the column names of the first query, ignoring column names from each subsequent query.
+
+The `UNION` command has a `UNION ALL` option. The difference between `UNION` and `UNION ALL` is that `UNION` will drop any duplicated rows in the result, whereas `UNION ALL` will simply append the complete results of one query to another.
```

Comment:
```suggestion
The `UNION` command has a `UNION ALL` option. The difference between `UNION` and `UNION ALL` is that `UNION` will drop any duplicated rows in the result, whereas `UNION ALL` will append the complete results of one query to another.
```

I would replace the text on line # 37 with this text to sort of connect/transition between the two snippets in the "Syntax" section.